src/scss/styles.scss

```diff
@@ -659,13 +659,15 @@ app-user-billing {
     }
 }
 
+$callout-bg: #fafafa;
```

Comment:
I'd prefer that we move to a standard button styling like the above screenshot and revert this.

Reply:
removed